packages/asc-ui/src/components/TextArea/TextAreaStyle.ts

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from 'styled-components'
+import { themeColor, focusStyleOutline } from '../../utils'
+
+type StyleProps = {
+  error?: string | boolean
+}
+
+const TextAreaStyle = styled.textarea<StyleProps>`
+  appearance: none;
+  font-size: 1rem;
+  border: solid 1px ${themeColor('tint', 'level5')};
+  border-radius: 0;
+  box-sizing: border-box;
+  line-height: 18px;
+  padding: 10px;
```

Comment:
For paddings and margins we use the `themeSpacing` util = (n * 4) :)

Reply:
Will change